From oliver@neukum.org  Thu Jul  3 13:09:29 2008
From: Oliver Neukum <oliver@neukum.org>
Date: Thu, 3 Jul 2008 17:14:16 +0200
Subject: USB: adding comment for ipaq forcing number of ports
To: Adrian Bunk <bunk@kernel.org>
Cc: Greg KH <greg@kroah.com>, linux-usb@vger.kernel.org
Message-ID: <200807031714.16617.oliver@neukum.org>
Content-Disposition: inline


The reason for forcing a number of ports should be documented.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ipaq.c |    5 +++++
 1 file changed, 5 insertions(+)

--- a/drivers/usb/serial/ipaq.c
+++ b/drivers/usb/serial/ipaq.c
@@ -570,6 +570,11 @@ static struct usb_serial_driver ipaq_dev
 	.description =		"PocketPC PDA",
 	.usb_driver = 		&ipaq_driver,
 	.id_table =		ipaq_id_table,
+	/*
+	 * some devices have an extra endpoint, which
+	 * must be ignored as it would make the core
+	 * create a second port which oopses when used
+	 */
 	.num_ports =		1,
 	.open =			ipaq_open,
 	.close =		ipaq_close,
